scripts/export-user-data.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import { semestersCollection } from './firebase-config';
+
+class Rating {
```

Comment:
Do you think making this a class is useful if it only has one method? Or are you planning on extending it with more in the future?